addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -66,11 +58,15 @@ def action_invite(self):
 
         mail_values = []
         for partner_id in self.partner_ids:
-            slide_channel_partner = self.channel_id._action_add_members(partner_id)
+            slide_channel_partner = self.channel_id._action_add_members(partner_id, add_member_mode='invite')
             if slide_channel_partner:
                 mail_values.append(self._prepare_mail_values(slide_channel_partner))
 
-        self.env['mail.mail'].sudo().create(mail_values)
+        created_mails = self.env['mail.mail'].sudo().create(mail_values)
+
+        # This will auto-publish the course if at least one email is sent and it was not published before
+        if created_mails and self.channel_id.can_publish and not self.channel_id.is_published:
```

Comment:
That is the autopublish feature we discussed with LNA not long ago, I thought we agreed on that? :p We could add a warning though. Is it not conceivable technically? We could just replace that with a warning somewhere, no autopublish at all. This would be worth discussing with Luc.